### PR TITLE
Update dependencies with darc

### PR DESCRIPTION
**Pull request by natemcmaster**
Manually ran `darc update-dependencies -c '.NET Core 3 Dev'`.

It missed:

* Updating DotNetEfPackageVersion, even though versions.details.xml was updated (cc @mmitche )
* Updating dependencies from aspnetcore-tooling (cc @pranavkm @rynowak )

### CODE REVIEW

File: src/Components/src/Microsoft.AspNetCore.Components.Build/targets/RazorCompilation.targets [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <GenerateRazorAssemblyInfo>false</GenerateRazorAssemblyInfo>
     <GenerateRazorHostingAssemblyInfo>false</GenerateRazorHostingAssemblyInfo>
     <GenerateProvideApplicationPartFactoryAttribute>false</GenerateProvideApplicationPartFactoryAttribute>
+    <_RazorComponentInclude>**\*.cshtml</_RazorComponentInclude>
```
Comment (dougbu) [inline src/Components/src/Microsoft.AspNetCore.Components.Build/targets/RazorCompilation.targets]: Which component uses this property?

**Comment by benaadams** [comment]
```
Middleware\HealthChecks.EntityFrameworkCore\src\
 DependencyInjection\EntityFrameworkCoreHealthChecksBuilderExtensions.cs(44,24): 

error CS1574: XML comment has cref attribute 'IDatabaseCreatorWithCanConnect'
 that could not be resolved 

[F:\vsagent\11\s\src\Middleware\HealthChecks.EntityFrameworkCore\src\Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore.csproj]

File: src/Components/src/Microsoft.AspNetCore.Components.Build/targets/RazorCompilation.targets (hunk 1 above)
Comment (natemcmaster) [reply src/Components/src/Microsoft.AspNetCore.Components.Build/targets/RazorCompilation.targets]: Not sure. I was just merging in https://github.com/aspnet/AspNetCore/pull/6713 which included the razor SDK update

**Comment by natemcmaster** [comment]
Sent email to figure out where that API went.

**Comment by benaadams** [comment]
```
System.Net.Http.HttpRequestException : Error while copying content to a stream.
---- System.IO.IOException : 
-------- System.TypeLoadException : Could not load type 'System.Buffers.OperationStatus'
 from assembly 'System.Memory, Version=4.2.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'.
```
Guessing you need a new runtime? https://github.com/dotnet/coreclr/pull/21912

**Comment by natemcmaster** [comment]
Progress:

* Updates from AspNetCore-Tooling are working now.
* DotnetEfPackageVersion is still skipped.
* The bot isn't opening PRs due to MAX_PATH (which should be addressed by #6698)

**Comment by natemcmaster** [comment]
@benaadams I ran update again to get the latest netcore.app. If the problem persists, then this will likely need to be sent to corefx/coreclr as a partner-blocking issues.

**Comment by benaadams** [comment]
Different one this time
```
 error : Missing Version metadata for the following external dependencies:
 error :  - Microsoft.JSInterop. 

2019-01-16T02:21:11.1735260Z F:\vsagent\8\s\build\repo.targets(270,5):
  [F:\vsagent\8\.dotnet\buildtools\korebuild\3.0.0-build-20190110.4\KoreBuild.proj]
2019-01-16T02:21:11.1735472Z F:\vsagent\8\s\build\repo.targets(270,5): 
 [F:\vsagent\8\.dotnet\buildtools\korebuild\3.0.0-build-20190110.4\KoreBuild.proj]
```

**Comment by natemcmaster** [comment]
The test failures are consistent, not just flaky. @ajcvickers were there any recent changes in EF Core which might lead to this error?

```
Microsoft.EntityFrameworkCore.Database.Command[20102]
      Failed executing DbCommand (19ms) [Parameters=[@p0='?' (DbType = Int32), @p1='?' (DbType = Int32), @p2='?' (DbType = Int32), @p3='?' (DbType = Decimal)], CommandType='Text', CommandTimeout='30']
      SET NOCOUNT ON;
      INSERT INTO [OrderDetails] ([AlbumId], [OrderId], [Quantity], [UnitPrice])
      VALUES (@p0, @p1, @p2, @p3);
      SELECT [OrderDetailId]
      FROM [OrderDetails]
      WHERE @@ROWCOUNT = 1 AND [OrderDetailId] = scope_identity();
System.Data.SqlClient.SqlException (0x80131904): The INSERT statement conflicted with the FOREIGN KEY constraint "FK_OrderDetails_Orders_OrderId". The conflict occurred in database "MusicStore", table "dbo.Orders", column 'OrderId'.
The statement has been terminated.
   at System.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__126_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.Internal.RelationalCommand.ExecuteAsync(IRelationalConnection connection, DbCommandMethod executeMethod, IReadOnlyDictionary`2 parameterValues, CancellationToken cancellationToken) in /_/src/EFCore.Relational/Storage/Internal/RelationalCommand.cs:line 340
```

**Comment by ajcvickers** [comment]
@AndriySvyryd Can you take a look at these failures?

**Comment by benaadams** [comment]
error :  - Microsoft.JSInterop again

```
2019-01-16T21:34:29.3946330Z F:\vsagent\12\s\build\repo.targets(270,5): error : Missing Version metadata for the following external dependencies:  [F:\vsagent\12\.dotnet\buildtools\korebuild\3.0.0-build-20190110.4\KoreBuild.proj]
2019-01-16T21:34:29.3946637Z F:\vsagent\12\s\build\repo.targets(270,5):
 error :  - Microsoft.JSInterop.  [F:\vsagent\12\.dotnet\buildtools\korebuild\3.0.0-build-20190110.4\KoreBuild.proj]
2019-01-16T21:34:29.3947862Z     0 Warning(s)
2019-01-16T21:34:29.3948286Z     1 Error(s)
```

**Comment by AndriySvyryd** [comment]
MusicStore fix in 8dbed6dedecf53f1ef81ac41690896b525fe7c69

**Comment by ajcvickers** [comment]
@AndriySvyryd Looks good. So this was broken by the temp key values change? /cc @divega

**Comment by AndriySvyryd** [comment]
@ajcvickers Yes

**Comment by divega** [comment]
@ajcvickers do you think there is anything actionable other than making sure we call out this breaking change when we ship the next preview?

**Comment by ajcvickers** [comment]
@divega I think we should discuss. We could consider a compat flag. I'll follow up.

**Comment by natemcmaster** [comment]
Build failures this time due to https://github.com/dotnet/arcade/issues/1677. The version of core-setup hasn't caught up with the version of corefx packages.

**Comment by benaadams** [comment]
Failure is unexpected assemblies
```
Starting test execution, please wait...
[xUnit.net 00:00:00.33]     Microsoft.AspNetCore.SharedFxTests.PlatformManifestListsAllFiles [FAIL]
Failed   Microsoft.AspNetCore.SharedFxTests.PlatformManifestListsAllFiles
Error Message:
 Assert.Empty() Failure
Collection: ["Microsoft.Win32.Registry", "System.Diagnostics.DiagnosticSource", "System.Reflection.Metadata", "System.Security.AccessControl", "System.Security.Cryptography.Cng", ...]
Stack Trace:
   at Microsoft.AspNetCore.SharedFxTests.PlatformManifestListsAllFiles() in /_/src/Framework/test/SharedFxTests.cs:line 72
Standard Output Messages:
 ==== file contents ====
...
*snip*

 ==== unexpected assemblies in the manifest ====
 Microsoft.Win32.Registry
System.Diagnostics.DiagnosticSource
System.Reflection.Metadata
System.Security.AccessControl
System.Security.Cryptography.Cng
System.Security.Principal.Windows


Results File: F:\vsagent\2\s\artifacts\logs\SharedFx-UnitTests-3.0.0-preview-19066-79.trx

Total tests: 4. Passed: 3. Failed: 1. Skipped: 0.
Test Run Failed.
Test execution time: 1.2700 Seconds
\vsagent\2\s\build\SharedFx.targets(104,5):
 error : SharedFx.UnitTests failed with exit code '1'. [F:\vsagent\2\.dotnet\buildtools\korebuild\3.0.0-build-20190110.4\KoreBuild.proj]

Build FAILED.
```

**Comment by natemcmaster** [comment]
Waiting on https://github.com/dotnet/core-setup/pull/4996. cc @dagood

**Comment by mikaelm12** [comment]
Hmm, took a look at the build failure and I haven't seen that before

**Comment by muratg** [comment]
@mikaelm12 can you investigate the signalr issue please?

**Comment by mikaelm12** [comment]
Yeah, will take a closer look

**Comment by mikaelm12** [comment]
Discussed with Nate. We think this is a possible regression in the new .Net Core runtime. Reaching out to the appropriate people.

**Comment by benaadams** [comment]
Might want to check corefx for issues on WaitForExit, see if there is anything related e.g. https://github.com/dotnet/corefx/issues/31713

**Comment by natemcmaster** [comment]
@ajaybhargavb found a workaround for https://github.com/dotnet/corefx/issues/31713. See https://github.com/aspnet/AspNetCore-Tooling/commit/2ad0641b1e8e0cebb17115e3b8a7eb878f460d5c#diff-fec826feae04e51c0d94076385408bdc

**Comment by BrennanConroy** [comment]
The redis stuff is a benign failure. Looks like the build failed because of some razor tests.
________________________________

**Comment by natemcmaster** [comment]
@muratg @Eilon one test failure in what appears to be a flaky test. Sounds sufficiently good to me to override the PR check. Thoughts?

**Comment by muratg** [comment]
@natemcmaster, agreed. :shipit:

**Comment by tmds** [comment]
```
2019-01-17T20:30:24.4873203Z   System.InvalidOperationException: No process is associated with this object.
2019-01-17T20:30:24.4873972Z      at System.Diagnostics.Process.EnsureState(State state)
2019-01-17T20:30:24.4874675Z      at System.Diagnostics.Process.EnsureState(State state)
2019-01-17T20:30:24.4875635Z      at System.Diagnostics.Process.GetWaitState()
2019-01-17T20:30:24.4876288Z      at System.Diagnostics.Process.WaitForExitCore(Int32 milliseconds)
2019-01-17T20:30:24.4876529Z      at System.Diagnostics.Process.WaitForExit()
```

@natemcmaster @benaadams this may help: you get this when you call `WaitForExit` after calling `Stop`/`Dispose` on a Process.

**Pull request merged**